app/MMB/src/scenes/list/cityImage/FromToRow.js

```diff
@@ -22,19 +21,15 @@ export const FromToRow = (props: Props) => (
   <View style={styles.row}>
     <View style={styles.flexItem}>
       <Text style={[styles.text, styles.cityText]}>
-        <Translation
-          passThrough={idx(props.departure, _ => _.airport.city.name)}
-        />
+        <Translation passThrough={props.departure.airport?.city?.name} />
```

Comment:
Would this print `null` if we do not have any fallback values?

Reply:
Indeed it would, I do think maybe we should handle this in the translation component rather than were ever we pass something to translation.

Reply:
fixed with #1313